src/core/packages/chrome/browser-internal/src/chrome_service.tsx

```diff
@@ -523,17 +515,41 @@ export class ChromeService {
         docLinks={docLinks}
         kibanaVersion={injectedMetadata.getKibanaVersion()}
         prependBasePath={http.basePath.prepend}
-        isSideNavCollapsed$={this.isSideNavCollapsed$}
-        toggleSideNav={setIsSideNavCollapsed}
       >
-        {includeSideNavigation ? getProjectSideNavComponent() : null}
+        <Router history={application.history}>
```

Comment:
Note: moved the Router rendering up from https://github.com/elastic/kibana/pull/229110/files#diff-333fe32fe36772f661ce2091cb24ec608f14efda629e44abd1ada1ae8d8ddb0aL261